UI/remote-text.cpp

```diff
@@ -88,6 +89,9 @@ void RemoteTextThread::run()
 
 		code = curl_easy_perform(curl.get());
 		if (code != CURLE_OK) {
+			blog(LOG_WARNING,
+			     "RemoteTextThread: Failed to upload file. %s",
+			     error);
```

Comment:
This should probably not include "upload file" in the warning text, as this function is used for both uploads and downloads (as an example, the Patreon box in about dialog).

Reply:
Ah, good catch. I'll fixed that.